scripts/rollup/shims/facebook-www/ReactDOMComponentTree_DO_NOT_USE.js

```diff
@@ -9,7 +9,7 @@
 
 const {
   __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED,
-} = require('ReactDOM-fb');
+} = require('ReactDOM');
```

Comment:
This file got deleted. Kill it here too?

Reply:
Meaning this shim.

Reply:
Good call. Done.